step.yml

```diff
@@ -1,14 +1,44 @@
 title: "Bitrise.io Cache:Push"
 summary: "Updates the Cache, if required"
 description: |-
-  This step checks whether the cache has to be updated (based on the
-  `Cache Paths` and `Ignore Paths from change check` inputs).
-  If it has to be updated, the step will create a new cache archive and upload it.
+  Checks whether cache has to be updated based on the **Cache Paths** and **Ignore Paths** inputs.
+  If it does, the Step creates a new cache archive and uploads it. 
+  It won't update/upload the cache if the cache hasn't changed nor in the case of a pull request unless you change the Step's `run_if` property. 
+  For caching, you will need the **Bitrise.io Cache:Pull** Step at the beginning of your Workflow (after the **Git Clone Repository** Step but before a dependency installer Step) to download the previous cache, if any. 
+  The **Bitrise.io Cache:Push** Step should be the very last Step in your Workflow.
+  
+  ### Configuring the Step
+  1. Cache any directory or file by adding it in the **Cache paths** field. 
```

Comment:
I suggest mentioning somewhere that the rest of the native Bitrise build and test steps are automatically configuring cache paths. So in the most basic case, there is no need for configuring this input, just adding the native build, test steps (android-build, ...) and cache pull, cache push steps and the caching will 'just work'.